docs/services/vision/segmentation.md

```diff
@@ -19,6 +19,7 @@ The types of segmenters supported are:
 
 - [**Obstacles point cloud (`obstacles_pointcloud`)**](#configure-an-obstacles_pointcloud-segmenter): A segmenter that identifies well-separated objects above a flat plane.
 - [**Object detector (`detector_3d_segmenter`)**](#configure-a-detector_3d_segmenter): This model takes 2D bounding boxes from an object detector and projects the pixels in the bounding box to points in 3D space.
+- [**Obstacles depth (`obstacles_depth`)**](#configure-an-obstacles_depth-segmenter): A segmenter for depth cameras that returns a projection of a bounding box or boxes with a Pose as a vector from the depth camera lens to the center of the obstacle identified.
```

Comment:
Can we change to "... that returns the obstacles as 3-dimensional boxes with a Pose as a vector.... "

Reply:
Done, perfect thank you!

Reply:
Wait I'm sorry I messed up. I'm actually wrong because in some cases (no intrinsics) this isn't true. I think the most accurate thing is "A segmenter for depth cameras that returns the perceived obstacles as a set of one or more GeometriesInFrame, each with a Pose as a vector from the depth camera lens to the center of the obstacle identified."

Reply:
This comment @sguequierre